sfe/templates/layout.html

```diff
@@ -1,16 +1,104 @@
-{{define "meta"}}
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <meta name="viewport" content="width=device-width, initial-scale=1">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    
-        <link rel="stylesheet" href="/static/main.css">
+{{define "header"}}
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Let's Encrypt - Self-Service Portal</title>
+    <link rel="icon" type="image/x-icon" href="/static/favicon.ico">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/purecss@3.0.0/build/pure-min.css"
+        integrity="sha384-X38yfunGUhNzHpBaEBsWLO+A0HDYOQi8ufWDkZ0k9e0eXz/tH3II7uKZ9msv++Ls" crossorigin="anonymous">
```

Comment:
It looks like we've got rules for most of what we need below; how much is pure-min.css still getting us? Does everything look horrendous if we don't pull it in?

Reply:
Yeah, I gave similar feedback to Phil. We should avoid third-party resources like this, and the use case is not strong enough to justify copying this CSS into our repository.

Reply:
I've gone ahead and pulled in the remaining heading, html, body, div, and button styles that the current pages rely on. It's now identical.